### DISCUSSION TITLE
‼️ Matching earnings claim bug

### DISCUSSION THREAD

**Discussion by Fillox112** [betfinio/public#9] [Bugs and Reports]
![photo_2024-09-24 13 19 27](https://github.com/user-attachments/assets/93a062af-fd71-4f42-9356-d43c576c9873)

Several users are experiencing issues with claiming BETs from binary matching. At the bottom, it says that the BETs will be credited in a few hours, but after the time runs out, nothing happens. The timer resets and starts counting down again, but there is nothing to claim.

**Comment by bf-monster** [comment]
Hello, thank you for report. We will investigate this issue

**Reply by bf-monster** [reply]
will be fixed in betfinio/affiliate#22

**Reply by Fillox112** [reply]
Is it gonna help you if I give you the wallet address of the accounts affected?

As far as I know, this issue has been happening for quite a long time. I remember reporting it at least 2-3 months ago on other accounts as well.

**Reply by bf-mike-mkt** [reply]
I am facing the same issue, tested today

**Reply by Fillox112** [reply]
I also got new people who can't claim their earnings. This needs to be resolved asap.

**Reply by bf-monster** [reply]
Fixed, should be working now

**Comment by bf-monster** [comment]
fixed
